src/controllers/update/secure.update.filter.controller.ts

```diff
@@ -12,4 +11,4 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
   postFilterPage(req, res, next, config.UPDATE_USE_PAPER_URL, getNextPage());
 };
 
-const getNextPage = () => isActiveFeature(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS) ? config.UPDATE_INTERRUPT_CARD_URL : config.UPDATE_ANY_TRUSTS_INVOLVED_URL;
+const getNextPage = () => config.UPDATE_INTERRUPT_CARD_URL;
```

Comment:
I think you can take this one a step further and replace the call to getNextPage() with config.UPDATE_INTERRUPT_CARD_URL, and then you can delete the getNextPage() function

Reply:
Simplified this.